### PR TITLE
Add custom login methods order

### LINKED ISSUE #24: add ordering of login methods on web3auth modal

### PR DESCRIPTION
**Pull request by lionellbriones**
Fixes #24

### CODE REVIEW

File: packages/ui/src/loginModal.ts [hunk 1]
```diff
@@ -254,8 +257,10 @@ export default class LoginModal extends SafeEventEmitter {
       this.hasSocialWallet = true;
       $socialLogins.classList.remove("w3a-group--social-hidden");
       const providerIcon = AllImages[`login-${method}${this.isDark && hasLightIcons.includes(method) ? "-light" : ""}`].image;
+      const orderIndex = this.loginMethodsOrder.indexOf(method) + 1;
+      const order = orderIndex || this.loginMethodsOrder.length + 1;
```
Comment (himanshuchawla009) [inline packages/ui/src/loginModal.ts:261]: isn't it will put all the methods which are not available in `loginMethodsOrder` at last (length + 1)?

Comment (lionellbriones) [reply packages/ui/src/loginModal.ts:261]: yes.. it will show what is specified in order, and the others will follow their default order

Comment (himanshuchawla009) [reply packages/ui/src/loginModal.ts:261]: let say `orderIndex` is undefined in line 260 for more than 1 method (say x methods) and let say `this.loginMethodsOrder.length` is 4 , then all x methods which are missing `orderIndex` will be added on 5th position.

**Review by himanshuchawla009 — APPROVED** [review]
lgtm

**Pull request merged**